codegen/src/attribute.rs

```diff
@@ -53,7 +53,7 @@ struct Step {
     /// reference.
     ///
     /// [`gherkin::Step`]: https://bit.ly/3j42hcd
-    step_arg_name: Option<syn::Ident>,
+    arg_name: Option<syn::Ident>,
```

Comment:
@barafael `step_` prefix was referring to a `gherkin::Step` here, rather than this struct itself.